blob/src/main/java/io/crate/blob/BlobTransferTarget.java

```diff
@@ -104,6 +104,8 @@ public void startTransfer(StartBlobRequest request, StartBlobResponse response)
 
         DigestBlob digestBlob = blobShard.blobContainer().createBlob(request.id(), request.transferId());
         digestBlob.addContent(request.content(), request.isLast());
+        blobShard.incrementTotalSize(digestBlob.size());
+        blobShard.incrementBlobsCount();
```

Comment:
I think it would make sense to have only one method that does both, as it imho doesn't make sense to call only one without calling the other.